docs/src/pages/roadmap.tsx

```diff
@@ -231,18 +234,31 @@ const roadmap: Item[] = [
 ];
 
 const milestones: Item[] = [
+  withRelease({
+    icon: mdiFaceRecognition,
+    title: 'Metadata Face Import',
+    description: 'Read face metadata in Digikam format during metadata extraction',
```

Comment:
Maybe just say during import

Reply:
Since this is currently [exclusive](https://github.com/immich-app/immich/releases#:~:text=Only%20face%20metadata%20in%20Digikam%20format%20is%20supported) to face metadata from Digikam, I thought it would be necessary to mention it in the roadmap for those who follow there.
what do you think?

Reply:
I think it is fine to add here like you have it, but maybe we just say "during import" instead of "during metadata extraction" which is a bit technical.

Reply:
:)